cmd/app.go

```diff
@@ -1280,6 +1282,23 @@ func askSubQuestion(q managementClient.SubQuestion) string {
 	return answer
 }
 
+func getAllDefaultAnswers(tv *managementClient.TemplateVersion, answers map[string]string) map[string]string {
+	for _, question := range tv.Questions {
+		if _, ok := answers[question.Variable]; !ok && checkShowIf(question.ShowIf, answers) {
+			answers[question.Variable] = question.Default
+			if checkShowSubquestionIf(question, answers) {
+				for _, subQuestion := range question.Subquestions {
+					// only ask the question if there is not an answer and it passes the ShowIf check
```

Comment:
Hmm, maybe this comment should be rewritten? We should only set the subquestion if it passes the showIf check? Asking the question sounds interactive and it appears this prompt should not be interactive?